### DISCUSSION TITLE
Applying advanced settings in VS2022 as settings.json can do in vscode?

### DISCUSSION THREAD

**Discussion by herkaz** [community/community#14809] [Copilot Conversations]
The few settings that can be configured in VS2022 for the copilot extension are saved in the private registry hive.

The .js files for copilot seem to check for all the same settings (example: "github.copilot.advanced.temperature"), but I don't know where I can put the settings so that they are honored. Is there a place in the registry hive that I should add keys?

**Comment by kzu** [comment]
The VS extension uses VS's own settings to control how things work on the VS/editor side. Not sure what .js files you're referring to, the VS extension doesn't use those. If those are Copilot for VS Code files, those aren't shared by the VS side. We are thinking of ways some settings that do apply across editors might be shared, but that's not supported at the moment.
